spec/views/hyrax/base/_relationships.html.erb_spec.rb

```diff
@@ -29,6 +30,7 @@
     let(:member_of_collection_presenters) { [collection] }
 
     before do
+      allow(controller).to receive(:current_user).and_return true
```

Comment:
I would have expected this to return `user` instead of true. (again at line 49)

Reply:
hmmm that looks like a copy/edit error... I had to make several adjustments to the search service and places using it to get it to always have the ability, so I had to change the specs several times. I'm not sure what was intended here.

Reply:
It was passing because it was only looking to see if we had a current user, not what the user was. But I changed it to return user so it's a more appropriate response.